lib/commands/update-kendoui.ts

```diff
@@ -62,13 +80,50 @@ class UpdateKendoUICommand implements ICommand {
 				return;
 			}
 
-			var packageIdx = parseInt(choice.packageIdx, 10)-1;
-			var downloadUri = packages[packageIdx].DownloadUrl;
 			this.updateKendoFiles(downloadUri).wait();
 
 		}).future<void>()();
 	}
 
+	private selectKendoVersion(packages: Server.IKendoDownloadablePackageData[]): IFuture<string> {
+		return ((): string => {
+			this.showTableWithAvailableVersions(packages);
+			var schema: IPromptSchema = {
+				type: "input",
```

Comment:
I've tried this, but the list is very long and in case you want to select one of the last packages (index 70 for example), it is not very useful. So I've decided to keep current code.